entity/ProductStoreEntities.xml

```diff
@@ -520,6 +523,32 @@ along with this software (see the LICENSE.md file). If not, see
         </seed-data>
     </entity>
 
+    <extend-entity entity-name="DataDocument" package="moqui.entity.document">
+        <relationship type="many" related="mantle.product.store.ProductStoreDataDocument" short-alias="storeDataDocuments">
+            <key-map field-name="dataDocumentId"/></relationship>
+    </extend-entity>
+    <entity entity-name="ProductStoreDataDocument" package="mantle.product.store">
+        <description>Relationship between the ProductStore and DataDocument Entities meant to replace ProductStore.productDataDocumentId fields and the like. See: https://forum.moqui.org/t/productstore-datadocument-options/303.</description>
+        <field name="productStoreId" type="id" is-pk="true"/>
+        <field name="storeDataDocumentTypeEnumId" type="id" is-pk="true"/>
```

Comment:
@jonesde Based on [this comment](https://github.com/moqui/mantle-udm/pull/92#discussion_r917292344) should the `storeDataDocumentTypeEnumId` field also be shortened to like `storeDataTypeEnumId`?

Reply:
Yes, very good point... maybe just typeEnumId to avoid having to trim down ProductStore and DataDocument in some way. There are a few others like this, FWIW.

Reply:
This should be done.